sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -234,7 +234,7 @@ def begin_recognize_business_cards(
 
         include_field_elements = kwargs.pop("include_field_elements", False)
 
-        if content_type is None:
+        if content_type is None and kwargs.get("continuation_token", None) is None:
```

Comment:
idk how much help it'll be now, but we can put this into a helper, something like
```python
content_type = kwargs.pop("content_type", None)
content_type = content_type_helper(content_type)
```